spec/src/main/asciidoc/core/packagingdeployment.asciidoc

```diff
@@ -1,6 +1,8 @@
 [[packaging_deployment]]
 
-== Packaging and deployment
+== Packaging and deployment in {cdi_full}
+
+This chapter is very similar to <<packaging_deployment_lite>> but describes the whole process for {cdi_full}. Certain parts may therefore overlap.
```

Comment:
I agree we shouldn't split this. That would become unbearable.

```suggestion
This chapter replaces <<packaging_deployment_lite>> for the purpose of {cdi_full}.
In case of overlap, the <<packaging_deployment_lite>> chapter should be considered merely informative.
```